src/domains/fdr-app/04_apim_fdr_fase1.tf

```diff
@@ -55,7 +55,7 @@ resource "azurerm_api_management_api_operation_policy" "fdr_pagopa_policy_nodoCh
   operation_id        = var.env_short == "d" ? "6218976195aa0303ccfcf902" : var.env_short == "u" ? "61e96321e0f4ba04a49d1286" : "61e9633dea7c4a07cc7d480e"
 
   #tfsec:ignore:GEN005
-  xml_content = templatefile("./api/fdr-fase1/nodoPerPa/v1/fdr_pagopa.xml.tpl", {
+  xml_content = templatefile("fdr_pagopa_nodoChiediFlussoRendicontazione.xml.tpl", {
```

Comment:
why remove context path `./api/fdr-fase1/nodoPerPa/v1/`

```
Invalid value for "path" parameter: no file exists at "fdr_pagopa_nodoChiediFlussoRendicontazione.xml.tpl";
```

Reply:
it was a typo, thanks for the insight